core/src/main/java/org/hibernate/ogm/datastore/document/impl/DotPatternMapHelpers.java

```diff
@@ -130,4 +132,15 @@ public static boolean organizeAssociationMapByRowKey(
 	private static MapStorageType getMapStorage(AssociationContext associationContext) {
 		return associationContext.getAssociationTypeContext().getOptionsContext().getUnique( MapStorageOption.class );
 	}
+
+	public static String getColumnSharedPrefixOfAssociatedEntityLink(AssociationKey associationKey) {
```

Comment:
Could you add a description to this method? Also I'd expect it to be used by some of the other dialects? I vaguely remember seeing it somewhere else before.